pkgs/applications/science/math/R/default.nix

```diff
@@ -25,7 +25,10 @@ stdenv.mkDerivation rec {
     ++ stdenv.lib.optionals stdenv.isDarwin [ Cocoa Foundation libobjc libcxx ]
     ++ stdenv.lib.optional javaSupport jdk;
 
-  patches = [ ./no-usr-local-search-paths.patch ];
+  patches = [
+    ./no-usr-local-search-paths.patch
+    ./isSymmetric-is-not-S4-generic-without-Matrix.patch
```

Comment:
Can we download that patch from somewhere (i.e. via `fetchpatch`) instead of checking it into the Nixpkgs repository?

Reply:
Sure. I got it from https://github.com/wch/r-source/commit/aeb75e12863019be06fe6c762ab705bf5ed8b38c.patch because I couldn't find a way to browse R's source (which is on subversion somewhere). Is that fine or do you want me to try to download it from R's official source?

Reply:
Downloading it from github would be fine!

Reply:
Done, thanks